subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -18,209 +18,23 @@ zephyr_library_sources_ifdef(
   crypto/crypto.c
   )
 
-if(CONFIG_BT_LL_SW_LEGACY)
-  zephyr_library_sources(
-    ll_sw/ctrl.c
-    ll_sw/ll.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_BROADCASTER
-    ll_sw/ll_adv.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_OBSERVER
-    ll_sw/ll_scan.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_CTLR_FILTER
-    ll_sw/ll_filter.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_CENTRAL
-    ll_sw/ll_master.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_CTLR_ADV_EXT
-    ll_sw/ll_adv_aux.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_HCI_MESH_EXT
-    ll_sw/ll_mesh.c
-    )
-  zephyr_library_sources_ifdef(
-    CONFIG_BT_CTLR_DTM
-    ll_sw/ll_test.c
-    )
-endif()
-
-if(CONFIG_BT_LL_SW_SPLIT)
-  zephyr_library_sources(
-    ll_sw/ull.c
```

Comment:
This (and other files) are common and should stay in the common CMakeLists.txt

Reply:
Right, I will move them to the common portion and resend.